src/liballoc/rc.rs

```diff
@@ -1715,8 +1715,8 @@ impl<T> Weak<T> {
 
     /// Consumes the `Weak<T>` and turns it into a raw pointer.
     ///
-    /// This converts the weak pointer into a raw pointer, preserving the original weak count. It
-    /// can be turned back into the `Weak<T>` with [`from_raw`].
+    /// This converts the weak pointer into a raw pointer, while still preserving the ownership of
+    /// one weak handle. It can be turned back into the `Weak<T>` with [`from_raw`].
```

Comment:
"Weak handle" is not terminology that is used in https://doc.rust-lang.org/1.45.0/std/rc/struct.Weak.html. Do you mean the same thing as weak reference?